src/pngimage.cpp

```diff
@@ -643,10 +643,9 @@ namespace Exiv2 {
                 if ( iccProfileDefined() ) {
                     DataBuf compressed;
                     if ( zlibToCompressed(iccProfile_.pData_,iccProfile_.size_,compressed) ) {
-
-                        const byte* nullComp = (const byte*) "\0\0";
-                        const byte*  type    = (const byte*) "iCCP";
-                        const uint32_t nameLength  = static_cast<uint32_t>(profileName_.size());
+                        const auto nullComp = (const byte*)"\0\0";
```

Comment:
😱 ! Maybe we should move these constant literals to the anonymous space and just have them like this?

```cpp
namespace {
const char* nullComp = "\0\0";
const char* type = "iCCP";
}

Note: I am not complaining about the change you did here, but just to the existing code.